app/reducers/index.js

```diff
@@ -167,7 +169,11 @@ export const galleryPictureSchema = new schema.Entity('galleryPictures', {
 export const gallerySchema = new schema.Entity('galleries');
 
 export const quoteSchema = new schema.Entity('quotes', {
-  comments: [commentSchema]
+  reactions: [reactionSchema]
```

Comment:
do I need this?

Reply:
I have not done extensive testing, yet removing it does not seem to do anything. I believe this is due to there not being a reaction field in the payload, it instead being named reactionsgrouped